File: PerlSolutions/EvenNumbers.pl
#Sample code to read in test cases:
open(INFILE, "<", $ARGV[0]) or die("Cannot open file $ARGV[0] for reading: $!");
while(my $line = <INFILE>) {
    next if($line =~ m/^\s$/);  # skip blank lines
    $line =~ s/(^\s|\s*$)//g;   # remove leading and trailing spaces
    if ($line % 2 == 0) {
      print "1\n";
    } else {
      print "0\n";
    }
}
close(INFILE);

File: PythonSolutions/EvenNumbers.py
import sys

with open(sys.argv[1], 'r') as test_cases:
    for test in test_cases:
        # ignore test if it is an empty line
        # 'test' represents the test case, do something with it
        if (int(test) % 2 == 0):
            print("1")
        else:
            print("0")
